pkg/kubelet/winstats/perfcounter_nodestats.go

```diff
@@ -217,15 +216,27 @@ func (p *perfCounterNodeStatsClient) getCPUUsageNanoCores() uint64 {
 }
 
 func getSystemUUID() (string, error) {
-	result, err := exec.Command("wmic", "csproduct", "get", "UUID").Output()
+	// Use PowerShell cmdlet "Get-CimInstance" instead of "wmic csproduct get
+	// uuid" for the following reasons:
+	// 1. The WMI command-line tool (Wmic) is deprecated. Use PowerShell cmdlets
+	// instead.
+	// https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/hh831568(v=ws.11)#wmi-providers
+	// 2. The PowerShell cmdlet "Get-CimObject" was removed from PowerShell 6
+	// and are no longer supported.
+	// https://github.com/MicrosoftDocs/PowerShell-Docs/issues/5156#issuecomment-558253083
+	command := "(Get-CimInstance -Class Win32_ComputerSystemProduct).UUID"
+	args := []string{"-NoProfile", "-NonInteractive", "-ExecutionPolicy", "Restricted", "-Command", command}
+	result, err := exec.Command("powershell.exe", args...).CombinedOutput()
 	if err != nil {
+		klog.Errorf("unable to get system uuid, reason: %q, detail: %q", err, result)
 		return "", err
 	}
-	fields := strings.Fields(string(result))
-	if len(fields) != 2 {
-		return "", fmt.Errorf("received unexpected value retrieving vm uuid: %q", string(result))
-	}
-	return fields[1], nil
+
+	systemUUID := strings.TrimRight(string(result), "\r\n")
+
+	klog.V(5).Infof("system uuid: %q", systemUUID)
```

Comment:
remove this since it's not necessary

Reply:
Thanks Andy, I'll remove the L237, but for L235 it should be remained since the PowerShell command result will be "65A08C83-1C96-D45A-DCAA-45504810B22A\r\n"